website/assets/site.css

```
@@ -0,0 +1,307 @@
+:root {
+  color-scheme: dark light;
+  --bg: #0b1020;
+  --panel: rgba(255, 255, 255, 0.07);
+  --panel-strong: rgba(255, 255, 255, 0.12);
+  --text: #f7f8fb;
+  --muted: #b6bed2;
+  --line: rgba(255, 255, 255, 0.14);
+  --accent: #84ccff;
+  --accent-strong: #38bdf8;
+  --shadow: 0 24px 80px rgba(0, 0, 0, 0.34);
+  font-family: Inter, ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", sans-serif;
```

Stylelint (17.10.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)